src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+// Actions
+const CHECK_STATUS = 'bookstore/categories/CREATE';
+
+// Reducer
+export const reducer = (state = [], action = {}) => {
+  switch (action.type) {
+    case CHECK_STATUS: {
```

Comment:
This reducer is required to be created by the redux toolkit using createSlice.

![image](https://user-images.githubusercontent.com/27709832/223594309-d35ebf79-a7a8-4507-a996-1614d4580cec.png)

Eg.

```
mport { createSlice } from "@reduxjs/toolkit"

export const booksSlice = createSlice({
    // your code
});
```
It should also be the same for books reducer.